src/components/ui4/ButtonsViewUi4.tsx

```diff
@@ -1,29 +1,35 @@
+/**
+ * IMPORTANT: Changes in this file MUST be synced between edge-react-gui and
+ * edge-login-ui-rn!
+ */
+
 import * as React from 'react'
 import { View, ViewStyle } from 'react-native'
-import Animated from 'react-native-reanimated'
 
-import { useFadeAnimation } from '../../hooks/animations/useFadeAnimation'
+import { EdgeAnim } from '../common/EdgeAnim'
+import { maybeComponent } from '../hoc/maybeComponent'
 import { styled } from '../hoc/styled'
 import { Space } from '../layout/Space'
 import { ButtonTypeUi4, ButtonUi4 } from './ButtonUi4'
 
 const INTER_BUTTON_SPACING_REM = 1
+const ANIM_DURATION = 1000
+const ANIM_DISTANCE_INCREMENT = 20
 
 export interface ButtonInfo {
   label: string
   onPress: () => void | Promise<void>
   disabled?: boolean
+  spinner?: boolean
+  testID?: string
+
+  animDistanceStart?: number
 }
 
 interface Props {
   // Specifies whether the component should be positioned absolutely.
   // Default value is false.
   absolute?: boolean
```

Comment:
Removed this fade prop because our animations effectively replace it, and it's
not used anyway.